packages/design-system/src/components/primitives/dnd/use-drop.ts

```diff
@@ -81,7 +85,11 @@ export const useDrop = <Data>(props: UseDropProps<Data>): UseDropHandlers => {
       if (fromCache !== undefined) {
         return fromCache;
       }
-      const result = getChildrenRects(parent);
+      const children =
+        latestProps.current.getValidChildren !== undefined
+          ? latestProps.current.getValidChildren(parent)
+          : parent.children;
```

Comment:
could simplify to this, right?
```suggestion
      const children = latestProps.current?.getValidChildren(parent) ?? parent.children
```

Reply:
ideally though getValidChildren is always defined and always returns children